run.cmd

```diff
@@ -17,12 +17,12 @@ if not defined VisualStudioVersion (
 set Platform=
 
 :: Restore the Tools directory
-call %~dp0init-tools.cmd
+call "%~dp0init-tools.cmd"
 if NOT [%ERRORLEVEL%]==[0] exit /b 1
 
 set _toolRuntime=%~dp0Tools
 set _dotnet=%_toolRuntime%\dotnetcli\dotnet.exe
 
 echo Running: %_dotnet% %_toolRuntime%\run.exe %*
-call %_dotnet% %_toolRuntime%\run.exe %*
+call "%_dotnet% %_toolRuntime%\run.exe" %*
```

Comment:
I think so. So could you please tell me should I edit this PR or create a new PR? I'm new here and don't know what's the correct way. Thanks for answering.